test/unit/menu.js

```diff
@@ -0,0 +1,272 @@
+// unit test t2 root menu object
+var sinon = require('sinon');
+var controller = require('../../lib/controller');
+var Tessel = require('../../lib/tessel/tessel');
+var Seeker = require('../../lib/discover.js');
+var util = require('util');
+var EventEmitter = require('events').EventEmitter;
+var logs = require('../../lib/logs');
+var Menu = require('terminal-menu');
+
+// template for creating fake tessels
+function createFakeTessel(options) {
+  var tessel = new Tessel({
+    connectionType: options.type || 'LAN',
+    authorized: options.authorized !== undefined ? options.authorized : true,
+    end: function() {
+      return Promise.resolve();
+    }
+
+  });
+
+  function serialNumber(options) {
+    if (options.type === 'USB') {
+      return options.serialNumber;
+    } else {
+      return false;
+    }
+  }
+  tessel.serialNumber = serialNumber(options);
+  tessel.name = options.name || 'a';
+
+  options.sandbox.stub(tessel, 'close', function() {
+    return Promise.resolve();
+  });
+  // return the fake tessel
+  return tessel;
+}
```

Comment:
This should be rolled into `TesselSimulator` https://github.com/tessel/t2-cli/blob/master/test/common/tessel-simulator.js#L4-L29